cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt

```diff
@@ -510,20 +602,36 @@ policy "pol_utilization" do
         label "Instance Id"
         path "dbInstanceIdentifier"
       end
-      field "engine" do
-        label "Engine"
+      field "platform" do
+        label "Platform"
+        path "databaseEngine"
       end
-      field "dbInstanceClass" do
+      field "resourceType" do
         label "Current Instance Class"
+        path "dbInstanceClass"
       end
-      field "recommended_size" do
+      field "newResourceType" do
         label "Recommended Instance Class"
+        path "recommended_size"
       end
       field "availabilityZone" do
         label "Availability Zone"
       end
-      field "percent_cpu_avg" do
+      field "cpuAverage" do
         label "CPU Average %"
+        path "percent_cpu_avg"
+      end
+      field "lookbackPeriod" do
+        label "Look Back Period"
```

Comment:
Same as changelog. For me it should be "Lookback Period" for this and all other labels.